### DISCUSSION TITLE
heal object failed. How to view details?

### DISCUSSION THREAD

**Discussion by beck-8** [minio/minio#18326] [Q&A]
16 sets of 35 14T hard drives
EC:4
I deliberately unmounted and formatted two disks in one of the servers and then remounted them. There are two errors when restoring data. There was no hardware damage prior to this.

My question is how to see the details of these two errors? Does this mean my data is lost?
If it is lost, can I see a clear reason and details of the lost data?
```
root@hostname:~# mc admin heal my
Objects Healed: 5,234, 162 TiB (31.1%)
Objects Failed: 2
Heal rate: 0 obj/s, 1.4 GiB/s
Estimated Completion: 2 days
```
is `mc admin heal my/ -r` ?

Thank you very much. Looking forward to your reply.

**Comment by khalilsarwari** [comment]
@harshavardhana @vadmeste can you please provide some guidance regarding this? any way to identify which paths and relaunch a heal attempt?
